### ISSUE TITLE
Remove custom masks from mod.

### ISSUE THREAD

**Issue by Ravicale** [species-nova/PD2HEAT#22]
The custom masks are an out-of-scope holdover from Restoration Mod and should be cut to reduce on filesize (bonus points for making a standalone version). `BlackMarketTweakData.lua` includes the build of the lua information for the masks, but the associated assets also need to be removed from the assets folder and any add xml file(s).

This needs to be done prior to release because it has the potential to break save files.
